pkg/query/config.go

```diff
@@ -39,3 +43,25 @@ func LoadConfigs(confYAML []byte) ([]Config, error) {
 	}
 	return queryCfg, nil
 }
+
+// BuildQueryConfig initializes and returns an Query client configuration from a static address.
```

Comment:
```suggestion
// BuildQueryConfig returns a query client configuration from a static address.
```

Essentially I don't think we initialize anything (:

Reply:
right, fixed